pkg/syncer/volume_health.go

```diff
@@ -116,12 +116,11 @@ func updateVolumeHealthStatus(ctx context.Context, k8sclient clientset.Interface
 	_, foundAnnHealthTS := pvc.Annotations[annVolumeHealthTS]
 	if !found || val != volHealthStatus || !foundAnnHealthTS {
 		// VolumeHealth annotation on pvc is changed, set it to new value
-		log.Debugf("updateVolumeHealthStatus: update volume health annotation for pvc %s/%s from old value %s to new value %s",
-			pvc.Namespace, pvc.Name, val, volHealthStatus)
 		metav1.SetMetaDataAnnotation(&pvc.ObjectMeta, annVolumeHealth, volHealthStatus)
 		timeNow := time.Now().Format(time.UnixDate)
 		metav1.SetMetaDataAnnotation(&pvc.ObjectMeta, annVolumeHealthTS, timeNow)
-		log.Debugf("updateVolumeHealthStatus: set annotation for health to %s at time %s for pvc %s/%s", volHealthStatus, timeNow, pvc.Namespace, pvc.Name)
+		log.Infof("updateVolumeHealthStatus: set volumehealth annotation for pvc %s/%s from old value %s to new value %s at time %s",
```

Comment:
Can you share the log for this line?
we are using %s specified for time type as well, just want to make sure it is printed properly. otherwise for non-string type can you use %v as a format specifier.

Reply:
This is the log

Reply:
our log message already have a timestamp - 2021-07-01T18:25:58.871Z
why do we need to print timestamps in the log?

Reply:
We have another annotation for volume health, annVolumeHealthTS which is the time stamp that health annotation is updated.
How about I change the log to something like this?
```
updateVolumeHealthStatus: set volumehealth annotation for pvc %s/%s from old value %s to new value %s and volumehealthTS annotation to %s
```